src/components/Create/components/pages/FundingCycles/FundingCyclesPage.tsx

```diff
@@ -93,9 +93,24 @@ export const FundingCyclesPage = () => {
               <Selection.Card
                 name="automated"
                 title={
-                  <Trans>
-                    Automated Funding Cycles <CreateBadge.Recommended />
-                  </Trans>
+                  <>
+                    <Trans>Automated Funding Cycles</Trans>{' '}
+                    <CreateBadge.Recommended
+                      tooltip={
+                        <span>
+                          <Trans>
+                            With Automated Funding Cycles, your funding
+                            configuration is locked for a set time period.
+                            <br />
+                            <br />
```

Comment:
**suggestion** might be nicer to wrap each text block in a `<p>`

(we should probably ban `br` in the codebase lol)